### PR TITLE
docs: fix useContractWrite export

### PR DESCRIPTION
**Pull request by aLIEzsss4**

### CODE REVIEW

File: packages/core/src/hooks/execute.ts [hunk 1]
```diff
@@ -124,7 +124,7 @@ function writeContract({
     if (calls === undefined) {
       throw new Error('No calls specified')
     }
-    const response = await account.execute(calls)
```
Comment (fracek) [inline packages/core/src/hooks/execute.ts:127]: In starknet.js v4 it's still called `execute`.

File: packages/core/src/hooks/starknetid.test.tsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ describe('useStarkName', () => {
     const keyPair = ec.getKeyPair(number.toBN(whitelistingPrivateKey))
     const signed = ec.sign(keyPair, hashed)
 
-    const { transaction_hash } = await account.execute([
```
Comment (fracek) [inline packages/core/src/hooks/starknetid.test.tsx:50]: This one should also stay as `execute`.

**Review by fracek — CHANGES_REQUESTED** [review]

**Comment by fracek** [comment]
Thanks. Added a small comment.

**Comment by aLIEzsss4** [comment]
> Thanks. Added a small comment.

Has been corrected the modifications, only the document has encountered errors.

**Review by fracek — APPROVED** [review]

**Comment by fracek** [comment]
Thank you for the fix!

**Pull request merged**